### DISCUSSION TITLE
Tailwindcss 3  can't  apply  changes on css classes  from  chrome devtool

### DISCUSSION THREAD

**Discussion by samehsy** [tailwindlabs/tailwindcss#6731] [Help] [answered]
Hi, I  am using   tailwind in my  **angular  13**  project  and  I used  to  change  CSS  classes like ( padding  and margins .....)  from  chrome  dev tools on DOM elements 
the  problem  which I  faced  is :  chrome  just applied the  first load  classes  and  ignore  any  changes  I  made  so  I  need  to  go back  to  my  code  and  changes  CSS classes  then  rebuild angular  to see  changes 

**Tailwindcss  :3.0.7** 
**postcss : 8.4.5**
**autoprefixer :10.4.0**

**Comment by LowNightSnack** [comment, marked as answer]
Aren't unused classes purged? So, the classes you are trying to call from the devtools do not exist in the first place.
